websites/T/Tiermaker/presence.ts

```diff
@@ -0,0 +1,92 @@
+let episode: string,
+    creator: string,
+    c_title: string,
+    c_category: string,
+    c_stateNum: number,
+    c_currentState: string;
+
+const presence = new Presence({
+    clientId: "919901175640358983"
+  }),
+
+  startStamp = Math.floor(Date.now() / 1000),
+  urlpath = document.location.pathname.split("/"),
+
+  getElement = (query: string): string | undefined => {
+    return document.querySelector(query)?.textContent;
+  },

```

Comment:
Might as well make use of arrow functions 
```suggestion
  getElement = (query: string) => document.querySelector(query)?.textContent,
```